scripts/export_to_s3.js

```diff
@@ -0,0 +1,200 @@
+var async = require('async');
+var Habitat = require('habitat');
+var request = require('request');
+var url = require('url');
+var AWS = require('aws-sdk');
+var env = require('../lib/environment');
+var mongoose = require( '../lib/mongoose' )(connectionReady);
+var Make = require( '../lib/models/make' )( mongoose.mongoInstance() );
+
+var documentCount;
+var aggregatedMakes = {};
+var aggregatedRedactedMakes = {};
+var currentSkipValue = 0;
+var aggregationLimit = 300;
```

Comment:
this seems pretty low; is each make actually 50KB? (16MB / 300 makes = 53.3KB/make). Or is this the only number that worked in practice?